legacy/logclient/persistance.go

```diff
@@ -0,0 +1,130 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package logclient
+
+import (
+	"fmt"
+	"sync"
+	"time"
+
+	"github.com/sirupsen/logrus"
+)
+
+// PersistentLoggingInfo holds peristand stats about the behavior of the auditor.
+// Since the auditor is a continuously running program, this data will begin
+// collection upon startup and lost on shutdown. Such data is only collected when
+// invoked with the --verbose flag.
+type PersistentLoggingInfo struct {
+	verboseLogging bool             // Determines if the user passed the --verbose flag, allowing persistent logs.
+	mutex          sync.Mutex       // Required to grab the queries after each measurement window.
+	measurement    QueryMeasurement // The measurement structure holding the amount of times GCR was queried.
+}
+
+// QueryMeasurement holds the logging information about number of queries per measurment window.
+type QueryMeasurement struct {
```

Comment:
The name `QueryMeasurement` sounds like a verb + noun, which is not good for a struct (it should ideally be just a noun or adjective+noun).

Reply:
Ack.